src/rpc.rs

```diff
@@ -22,6 +22,24 @@ impl RPC {
     ) -> Router<MissionServer<RPC>, transport::server::Unimplemented> {
         Server::builder().add_service(MissionServer::new(RPC { ipc }))
     }
+
+    pub async fn request<P, R>(&self, method: &str, params: Option<P>) -> Result<R, Status>
+    where
+        P: serde::Serialize + Send + Sync + 'static,
+        for<'de> R: serde::Deserialize<'de> + Send + Sync + std::fmt::Debug + 'static,
+    {
+        self.ipc.request(method, params).await.map_err(to_status)
+    }
+
+    pub async fn notification<P>(&self, method: &str, params: Option<P>) -> Result<(), Status>
+    where
+        P: serde::Serialize + Send + Sync + 'static,
+    {
+        self.ipc
+            .notification(method, params)
+            .await
+            .map_err(to_status)
+    }
```

Comment:
The only reason for these helpers is to not have to map the error type for each RPC method below